TESTS/netsocket/tcp/main.cpp

```diff
@@ -214,8 +225,10 @@ Case cases[] = {
     Case("TCPSOCKET_BIND_WRONG_TYPE", TCPSOCKET_BIND_WRONG_TYPE),
     Case("TCPSOCKET_BIND_UNOPENED", TCPSOCKET_BIND_UNOPENED),
     Case("TCPSOCKET_SETSOCKOPT_KEEPALIVE_VALID", TCPSOCKET_SETSOCKOPT_KEEPALIVE_VALID),
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE != MESH
     Case("TCPSOCKET_RECV_100K", TCPSOCKET_RECV_100K),
     Case("TCPSOCKET_RECV_100K_NONBLOCK", TCPSOCKET_RECV_100K_NONBLOCK),
+#endif
```

Comment:
I don't understand why this makes sense?

Does it really help if we start skipping test cases?

Reply:
Its a big challenge to pass  100k recv  in mesh.
After increasing timeouts up to 20 minutes  tests mostly pass sometimes fails.
Smaller timeouts causes 100% fails
Its  rfc864  tests  so receiving DUT  cant slow down transmision if mesh network is unable to deliver  packets on time. So problem with TCP retransmision traffic will arise.
Maybe the solution it to keep this increased timeouts for mesh and run tests only in RF box  under no other devices "RF traffic "and "clear channel" condition.

Reply:
Test skip reverted. It is successful under  local mesh BR. Testing mesh performance under heavy RF traffic is beyond the scope of netsockets tests.